fs/walk_test.ts

```diff
@@ -341,3 +341,16 @@ testWalk(
   // TODO(kt3k): Enable this test
   true,
 );
+
+testWalk(
+  (d: string) => {
+    return Deno.run({cmd:['mkfifo', `${d}/mkfifo`]});
+  },
+  async function fifoSocket(listener: Deno.Process) {
+    assertReady(1);
+    const files = await walkArray(".", { followSymlinks: true });
+    assertEquals(files, [".", "mkfifo"]);
```

Comment:
I guess I have to revert back to expecting only ['.'] as #1792 ignores everything but files, dirs and symlinks iirc.